apps/crn-server/src/app.ts

```diff
@@ -259,6 +270,8 @@ export const appFactory = (libs: Libs = {}): Express => {
   const userController =
     libs.userController || new Users(userDataProvider, assetDataProvider);
   const labsController = libs.labsController || new Labs(squidexGraphqlClient);
+  const workingGroupsController =
+    libs.workingGroupsController || new WorkingGroups(workingGroupDataProvider);
```

Comment:
really small one, but maybe all variables should be in singular or plural, like

`workingGroupsController` / `workingGroupsDataProvider`

or 

`workingGroupController` / `workingGroupDataProvider`


I always have a hard time to find using ctrl + F when the variables are mixed in singular and plural 😅